support-frontend/app/controllers/Application.scala

```diff
@@ -132,10 +130,7 @@ class Application(
     val guestAccountCreationToken = request.flash.get("guestAccountCreationToken")
 
     implicit val settings: AllSettings = settingsProvider.getAllSettings()
-    request.user.traverse[Attempt, IdUser](user => identityService.getUser(user.minimalUser)).fold(
```

Comment:
we don't need to getUser again as we already got it as part of the maybe authenticated action